docs/GettingStarted.md

```diff
@@ -4,14 +4,16 @@
 
 This is a targeted summary of setup React Native setup for specifically React Native Windows. See the [React Native Getting Started Guide](http://facebook.github.io/react-native/docs/getting-started.html) for full details.
 
-#### Dependencies: Node, Python, JDK
+*Note*: The Linux commands in the offical React Native Getting Started Guide should work on the Windows Subsytem for Linux bash shell, but this scenario is not currently officially supported. PowerShell or the Visual Studio command prompt are recommended.
 
```

Comment:
`Visual Studio command prompt` -> `Developer Command Prompt` would be more accurate.

Reply:
fixed to Visual Studio Developer Command Prompt!